layers/+misc/nlinum/funcs.el

```diff
@@ -0,0 +1,20 @@
+;;; packages.el --- nlinum Layer packages File
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Thomas de Beauchêne <thomas.de.beauchene@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defun spacemacs/nlinum-on ()
```

Comment:
Misleading name, because it doesn't always enable `nlinum-mode`. Better name: `spacemacs/nlinum-maybe-on`. (there are plenty of "maybe" functions in the ecosystem, e.g. `ahs-mode-maybe` and `doc-view-mode-maybe`)